src/drivers/batt_smbus/parameters.c

```diff
@@ -55,10 +55,11 @@ PARAM_DEFINE_FLOAT(BAT_C_MULT, 1.0f);
  *
  * @reboot_required true
  * @min 0
- * @max 2
- * @group Sensors
- * @value 0 AutoDetect
+ * @max 3
+ * @value 0 Auto(BQ40Z50)
  * @value 1 BQ40Z50 based
  * @value 2 BQ40Z80 based
+ * @value 3 BQ78350 based
+ * @group Sensors
  */
-PARAM_DEFINE_INT32(BAT_SMBUS_MODEL, 0);
+PARAM_DEFINE_INT32(BAT_SMBUS_MODEL, 3);
```

Comment:
Do we want to make BMS default in PX4?

Reply:
It is not making BMS default, only choosing the default board config if batt_smbus is enabled. The default previously was 0(auto) which was not even implemented, selecting MODEL 1 anyway.